torch/_dynamo/test_case.py

```diff
@@ -98,7 +104,64 @@ def tearDown(self) -> None:
 
 
 class CPythonTestCase(TestCase):
+    """
+    Enable certain features that are off by default (i.e. tracing through unittest)
+    """
+
     _stack: contextlib.ExitStack
+    dynamo_strict_nopython = True
+
+    assertEqual = unittest.TestCase.assertEqual  # type: ignore[assignment]
```

Comment:
Dynamo's `TestCase` class overrides some of these methods and it makes it hard to trace. I added these methods back so that executing the code with and without dynamo uses the same methods.

Reply:
Can you add a comment in the code about that?